tasks/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: rh-sign-image-cosign
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task to sign container images in snapshot by cosign
+  params:
+    - name: snapshotPath
+      description: Path to the JSON string of the mapped Snapshot spec in the data workspace
+      type: string
+    - name: secretName
+      description: Name of secret containing needed credentials
+      type: string
+  workspaces:
+    - name: data
+      description: Workspace to read and save files
+  steps:
+    - name: sign-image
+      image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
+      env:
+        - name: AWS_REGION
+          valueFrom:
+            secretKeyRef:
+              name: $(params.secretName)
+              key: AWS_REGION
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              name: $(params.secretName)
+              key: AWS_ACCESS_KEY_ID
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: $(params.secretName)
+              key: AWS_SECRET_ACCESS_KEY
+        - name: SIGN_KEY
+          valueFrom:
+            secretKeyRef:
+              name: $(params.secretName)
+              key: SIGN_KEY
+      script: |
+        #!/usr/bin/env bash
+        set -eu
```

Comment:
I think here we can actually also include `-x`, so `set -eux`. The main reason we refrain from `-x` sometimes is to avoid leaking of secrets to the log, but here it should be ok, right? The only thing that will be printed in this task is the value of `SIGN_KEY`, but that's just the name of the key, right? Not the actual key value. And all the `AWS_` secrets are passed as env vars, so they will not be printed, so it should be fine. WDYT?

Reply:
SIGN_KEY can be printed. Added -x.

Reply:
Great, thanks.